Source/WarpX.cpp

```diff
@@ -2133,6 +2150,10 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
     AllocInitMultiFab(Efield_fp[lev][1], amrex::convert(ba, Ey_nodal_flag), dm, ncomps, ngEB, tag("Efield_fp[y]"));
     AllocInitMultiFab(Efield_fp[lev][2], amrex::convert(ba, Ez_nodal_flag), dm, ncomps, ngEB, tag("Efield_fp[z]"));
 
+    AllocInitMultiFab(Afield_fp[lev][0], amrex::convert(ba, Ax_nodal_flag), dm, ncomps, ngEB, tag("Afield_fp[x]"));
+    AllocInitMultiFab(Afield_fp[lev][1], amrex::convert(ba, Ay_nodal_flag), dm, ncomps, ngEB, tag("Afield_fp[y]"));
+    AllocInitMultiFab(Afield_fp[lev][2], amrex::convert(ba, Az_nodal_flag), dm, ncomps, ngEB, tag("Afield_fp[z]"));
+
```

Comment:
Similar comment as above, these allocations will probably go into a member function `LaserEnvelope::AllocateLevelMFs` in the file `LaserEnvelope.cpp`, similarly to what is currently done for the class `HybridPICModel`.
```suggestion
```